dash/development/base_component.py

```diff
@@ -311,6 +367,139 @@ def __len__(self):
         return length
 
 
+def schema_is_nullable(type_object):
+    if type_object and 'name' in type_object:
+        if type_object['name'] == 'enum':
+            values = type_object['value']
+            for v in values:
+                value = v['value']
+                if value == 'null':
+                    return True
+        elif type_object['name'] == 'union':
+            values = type_object['value']
+            if any(schema_is_nullable(v) for v in values):
+                return True
+    return False
+
+
+def js_to_cerberus_type(type_object):
+    def _merge(x, y):
+        z = x.copy()
+        z.update(y)
+        return z
+
+    def _enum(x):
+        schema = {'allowed': [],
+                  'type': ('string', 'number')}
+        values = x['value']
+        for v in values:
+            value = v['value']
+            if value == 'null':
+                schema['nullable'] = True
+                schema['allowed'].append(None)
+            elif value == 'true':
+                schema['allowed'].append(True)
+            elif value == 'false':
+                schema['allowed'].append(False)
+            else:
+                string_value = v['value'].strip("'\"'")
+                schema['allowed'].append(string_value)
+                try:
+                    int_value = int(string_value)
+                    schema['allowed'].append(int_value)
+                except ValueError:
+                    pass
+                try:
+                    float_value = float(string_value)
+                    schema['allowed'].append(float_value)
+                except ValueError:
+                    pass
+        return schema
+
+    converters = {
+        'None': lambda x: {},
+        'func': lambda x: {},
+        'symbol': lambda x: {},
+        'custom': lambda x: {},
+        'node': lambda x: {
+            'anyof': [
+                {'type': 'component'},
+                {'type': 'boolean'},
+                {'type': 'number'},
+                {'type': 'string'},
+                {
+                    'type': 'list',
+                    'schema': {
+                        'type': (
+                            'component',
+                            'boolean',
+                            'number',
+                            'string')
+                    }
+                }
+            ]
+        },
+        'element': lambda x: {'type': 'component'},
+        'enum': _enum,
+        'union': lambda x: {
+            'anyof': [js_to_cerberus_type(v) for v in x['value']],
+        },
+        'any': lambda x: {
+            'type': ('boolean',
+                     'number',
+                     'string',
+                     'dict',
+                     'list')
+        },
```

Comment:
Shouldn't any just return `True` validated for any prop type.

Reply:
Yeah, I think I misread the definition of `PropTypes.any`. Will change